cmd/dep/testdata/harness_tests/init/manifest-exists/final/manifest.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
I thought it was probably best to keep this just to make sure that the manifest didn't change as it shouldn't have in this scenario.